recipes/norm/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "cci.20210118":
```

Comment:
We strongly prefer only having official release when possible... could you please add this version ?https://github.com/USNavalResearchLaboratory/norm/releases/tag/v1.5.8

Reply:
That version won't work (for me) as it has no cmake, but waf as primary build system. I will ask the autthor for a new release.